.github/workflow/ci-cd.yml

```
@@ -0,0 +1,102 @@
+
+      - name: Process and Upload Files to S3
+        env:
+          BUCKET: see-project
+          SRC_DIR: dist
+          FINAL_DIR: dist_final
+          COMPRESSED_LIST: compressed_files.txt
+        run: |
+          set -euo pipefail
+          rm -rf "$FINAL_DIR" "$COMPRESSED_LIST"
+          mkdir -p "$FINAL_DIR"
+          while IFS= read -r file; do 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)